### ISSUE TITLE
Train Pre-Experienced Model

### ISSUE THREAD

**Issue by Schopenhauer-loves-Hegel** [tianyi-lab/Superfiltering#3]
So this method ignore `Train Pre-Experienced Model` step mentioned in the previous paper [From Quantity to Quality: Boosting LLM Performance with Self-Guided Data Selection for Instruction Tuning](https://github.com/tianyi-lab/Cherry_LLM). In the previous paper, the pre-finetuning step is deemed rather important. May I inquire if this step was removed from the repository and the paper due to its negligible impact?

**Comment by MingLiiii** [comment]
Thank you so much for your interest in our work!

**Direct answer of your question**: The pre-finetuning step is kept in the codes and paper of the [Cherry_LLM](https://github.com/tianyi-lab/Cherry_LLM). It is removed in the codes and paper in the of the [Superfiltering](https://github.com/tianyi-lab/Superfiltering). 

**Main Differences**: In the paper [From Quantity to Quality: Boosting LLM Performance with Self-Guided Data Selection for Instruction Tuning](https://github.com/tianyi-lab/Cherry_LLM), we calculate the conditioned Loss/PPL in IFD scores with the [instruction-response template](https://github.com/tianyi-lab/Cherry_LLM?tab=readme-ov-file#prompt), thus LLMs need to be equipped with the basic instruction-following ability to get used to the template. However, in the paper [Superfiltering: Weak-to-Strong Data Filtering for Fast Instruction-Tuning](https://arxiv.org/abs/2402.00530), we optimized this process by discarding the template when calculating the conditioned Loss/PPL, which is important for the consistency between small and large language models. This difference is directly shown in the code. 

We will add this to the main pages of our repos in the next version.

**Comment by Schopenhauer-loves-Hegel** [comment]
Thank you for your reply! I would like to clarify my understanding.

If I have a raw dataset that consists of `prompt-response` pairs and I don't want to use any templates, can I simply use the superfilter method to assess the data quality without going through the step of training a pre-experienced model?

**Comment by MingLiiii** [comment]
Yes, you are correct!
That is exactly why we proposed Superfiltering, which is much simpler and faster.
